tests/unit/Messaging/MessageBufferTest.php

```diff
@@ -294,7 +294,7 @@ static function (Frame $frame): void {},
             false,
             null,
             0,
-            0x8000000000000000
+            -1
```

Comment:
At least on my system, `0x8000000000000000` results in a float that fails the type check for this param (`int|null`).  Since the class also rejects negative numbers, I changed the test to cover that condition since the engine will cover the non-integer case.